plugins/discourse-narrative-bot/assets/javascripts/initializers/new-user-narrative.js.es6

```diff
@@ -19,12 +19,12 @@ function initialize(api) {
   api.modifyClass("controller:topic", {
     pluginId: PLUGIN_ID,
 
-    _togglePostBookmark(post) {
+    _modifyBookmark(bookmark, post) {
```

Comment:
@martin-brennan blast from the past. this was renamed, but I don't see any method with that name which this would override, or anything calling this directly. Is that name correct?

Reply:
Oh that's very odd...it should be `_modifyPostBookmark` I think. I probably changed this halfway through this PR then changed again and didn't update this part. Not sure if the impact of this on `new-user-narrative-bot`...